ddtrace/internal/writer.py

```diff
@@ -271,67 +270,57 @@ def _log_error_status(self, response):
             )
 
 
-class Q(Queue):
-    """
-    Q is a threadsafe queue that let's you pop everything at once and
-    will randomly overwrite elements when it's over the max size.
-
-    This queue also exposes some statistics about its length, the number of items dropped, etc.
-    """
-
+class TraceQueue:
```

Comment:
```suggestion
class TraceQueue(object):
```

Python 2 requires that to not use old-style classes.